src/containers/guest-home-page/login-dialog/LoginDialog.jsx

```diff
@@ -20,8 +20,8 @@ const LoginDialog = () => {
   const { setAlert } = useSnackBarContext()
   const [loginUser] = useLoginMutation()
 
-  const { handleSubmit, handleInputChange, handleBlur, data, errors } = useForm(
-    {
+  const { handleSubmit, handleInputChange, handleBlur, data, errors, isDirty } =
+    useForm({
```

Comment:
It will be more readable

```suggestion
  const {
    handleSubmit,
    handleInputChange,
    handleBlur,
    data,
    errors,
    isDirty
   } = useForm({
```